diff --git a/vscode/build/gulpfile.extensions.js b/vscode/build/gulpfile.extensions.js
index 32a66ab..ddc3ff1 100644
--- a/vscode/build/gulpfile.extensions.js
+++ b/vscode/build/gulpfile.extensions.js
@@ -66,7 +66,22 @@ const compilations = [
 	'extensions/typescript-language-features/tsconfig.json',
 	'extensions/vscode-api-tests/tsconfig.json',
 	'extensions/vscode-colorize-tests/tsconfig.json',
-	'extensions/vscode-test-resolver/tsconfig.json'
+	'extensions/vscode-test-resolver/tsconfig.json',
+	'extensions/custom.codex-project-manager/tsconfig.json',
+	'extensions/custom.codex-project-manager/webviews/tsconfig.json',
+	'extensions/custom.codex-chat-and-comments/tsconfig.json',
+	'extensions/custom.codex-chat-and-comments/webviews/codex-webviews/tsconfig.json',
+	'extensions/custom.codex-scripture-viewer/tsconfig.json',
+	'extensions/custom.codex-scripture-viewer/webviews/codex-webviews/tsconfig.json',
+	'extensions/custom.codex-editor-extension/tsconfig.json',
+	'extensions/custom.codex-editor-extension/webviews/codex-webviews/tsconfig.json',
+	'extensions/custom.codex-editor-extension/webviews/dictionary-side-panel/tsconfig.json',
+	'extensions/custom.codex-editor-extension/webviews/editable-react-table/tsconfig.json',
+	'extensions/custom.codex-editor-extension/webviews/obs/tsconfig.json',
+	'extensions/custom.codex-editor-extension/webviews/usfm-viewer/tsconfig.json',
+	'extensions/custom.shared-state-store/tsconfig.json',
+	'extensions/custom.pythoninstaller/tsconfig.json',
+	'extensions/custom.python/tsconfig.json'
 ];
 
 const getBaseUrl = out => `https://ticino.blob.core.windows.net/sourcemaps/${commit}/${out}`;
